shared/qt/wrappers/qt-wrappers.cpp

```diff
@@ -318,10 +318,10 @@ void SetComboItemEnabled(QComboBox *c, int idx, bool enabled)
 			       : Qt::NoItemFlags);
 }
 
-void setThemeID(QWidget *widget, const QString &themeID)
+void setClasses(QWidget *widget, const QString &newClasses)
```

Comment:
Do we want to go with `setClasses` or something similar to CSS' `classList` which have `add` and `remove` utility functions with duplicate-detection? Feels a bit weird to me as a CSS user to completely purge the existing class without caring what the existing contents were.

Reply:
I did something like that with the helper functions on #8447
https://github.com/obsproject/obs-studio/pull/8447/files#diff-1e0673933828dd8c9df583b80101ad9dfb838e9fefd8ad0aec4d28aae92d3bd5R91

I opted not to overhaul things on this particular PR but I think that is the road we should go down, yes